test/evergreen/setup_spawn_host.sh

```diff
@@ -23,6 +23,21 @@ if [[ -n $WT_ARCHIVE ]]; then
     tar --wildcards -xzf $WT_ARCHIVE
 fi
 
+# Setup the gdb environment if there are core dumps present in the artefacts.
```

Comment:
As discussed:
```
grep SOURCE_DIR CMakeCache.txt
WiredTiger_SOURCE_DIR:STATIC=/home/ubuntu/src/wiredtiger
```
This entry in the `CMakeCache.txt` file gives you the source directory and will allow you to setup gdb in the absence of a core file. This allows for the possibility of having gdb ready to run if to inspect test exection as needed.

Reply:
Thank you that is so much more clearer! [275063e](https://github.com/wiredtiger/wiredtiger/pull/9556/commits/275063e6f212a378f66a4a0e54fdb786f7b8403d)